packages/react-components/react-table/stories/DataGrid/KeyboardColumnResizing.stories.tsx

```diff
@@ -187,25 +221,18 @@ export const KeyboardColumnResizing = () => {
       getRowId={item => item.file.label}
       selectionMode="multiselect"
       resizableColumns
-      columnSizingOptions={{
-        file: {
-          minWidth: 80,
-          defaultWidth: 120,
-        },
-        author: {
-          defaultWidth: 180,
-          minWidth: 120,
-          idealWidth: 180,
-        },
-      }}
+      columnSizingOptions={columnSizingOptions}
     >
       <DataGridHeader>
         <DataGridRow selectionCell={{ 'aria-label': 'Select all rows' }}>
           {({ renderHeaderCell, columnId }, dataGrid) =>
             dataGrid.resizableColumns ? (
               <Menu openOnContext>
                 <MenuTrigger>
-                  <DataGridHeaderCell ref={el => (refMap.current[columnId] = el)}>
+                  <DataGridHeaderCell
+                    ref={el => (refMap.current[columnId] = el)}
+                    className={currentlyResizing === columnId ? styles.currentlyResizing : undefined}
```

Comment:
instead of applying this manually, couldn't the DataGrid just support this by applying some kind of data-* attribute that the DataGrid could style?

Reply:
Thats a great feedback, I'll explore this option.